### ISSUE TITLE
Asset validation failed - Error while archiving and distributing the app with FBAudienceNetwork

### ISSUE THREAD

**Issue by jkozlowicz** [facebook/facebook-ios-sdk#2318]
### Checklist before submitting a bug report

- [X] I've updated to the latest released version of the SDK
- [X] I've searched for existing [GitHub issues](https://github.com/facebook/facebook-ios-sdk/issues)
- [X] I've looked for existing answers on [Stack Overflow](https://facebook.stackoverflow.com), the [Facebook Developer Community Forum](https://developers.facebook.com/community/) and the [Facebook Developers Group](https://www.facebook.com/groups/fbdevelopers)
- [X] I've read the [Code of Conduct](https://github.com/facebook/facebook-ios-sdk/blob/main/CODE_OF_CONDUCT.md)
- [X] This issue is not security related and can safely be disclosed publicly on GitHub

### Xcode version

15.0

### Facebook iOS SDK version

16.3.1

### Dependency Manager

SPM

### SDK Framework

Core

### Goals

I want to monetise my app with FBAudienceNetwork ads. To do that I followed this guide:
https://developers.facebook.com/docs/audience-network/setting-up/platform-setup/ios/add-sdk/
to add FBAudienceNetworkSDK to my iOS app

### Expected results

My app can be properly built and distributed with FBAudienceNetworkSDK.

### Actual results

My app cannot be properly built and distributed with FBAudienceNetworkSDK, because there seems to be minimum OS version mismatch between the FBAudienceNetworkSDK and my app. There is no problem between my app and the general FB SDK, however. Building and running locally works, but whenever I tried to archive and distribute the app, I get the following error:

<img width="740" alt="Screenshot 2023-12-29 at 15 19 32" src="https://github.com/facebook/facebook-ios-sdk/assets/4753742/b294d445-105c-4692-84a6-c443167cd9d0">

### Steps to reproduce

- Follow this guide: https://developers.facebook.com/docs/audience-network/setting-up/platform-setup/ios/add-sdk/
- Download [iOS Audience Network SDK 6.14.0](https://developers.facebook.com/resources/FBAudienceNetwork-6.14.0.zip)
- Unzip the SDK
- Copy the contents of the `Dynamic` directory to your app
- Archive the project

### Code samples & details

```swift
import FBAudienceNetwork
```

**Comment by sam-bensley** [comment]
Hey did you ever work out why this was happening?

**Comment by jkozlowicz** [comment]
@sam-bensley nope, but restarting the mac and re-adding the facebook SDK solved the problem for me.

**Comment by Zhiyi-Zhang** [comment]
Hi! After reviewing this issue, we're closing it because it relates to **FBAudienceNetwork**, which is a separate closed-source binary SDK not maintained in this repository (facebook-ios-sdk).

The Facebook iOS SDK and FBAudienceNetwork are distributed independently — this repo only contains the source code for FBSDKCoreKit, FBSDKLoginKit, FBSDKShareKit, and related open-source modules.

For FBAudienceNetwork issues, please report through the [Meta Business Help Center](https://www.facebook.com/business/help) or the [Meta for Developers](https://developers.facebook.com/support/) support channel.

We're closing this to keep the issue tracker focused on the open-source SDK components maintained here. Sorry for the inconvenience!
